handle/UnitGroup.lua

```diff
@@ -0,0 +1,21 @@
+require('CeresStdLib.base.Native')
+
+local __u
+
+local function tempEnumUnit()
+    return __u
+end
+
+replaceNative('GetEnumUnit', tempEnumUnit)

```

Comment:
=> `replaceNative('getenumunit', function() return __u end)`?

Reply:
Okay. Will inline that.